lucene/core/src/java/org/apache/lucene/store/IndexInput.java

```diff
@@ -98,7 +98,7 @@ public IndexInput clone() {
   
   /**
    * Creates a slice of this index input, with the given description, offset, and length. 
-   * The slice is seeked to the beginning.
+   * The slice is sought to the beginning.
```

Comment:
I’m wondering if this is intentional to refer to the `seek` method

Reply:
Oh, probably sought! Will fix it too.